cmd/activator/main.go

```diff
@@ -76,20 +76,20 @@ func main() {
 	}
 	servingClient, err := clientset.NewForConfig(clusterConfig)
 	if err != nil {
-		logger.Fatal("Error building serving clientset: %v", zap.Error(err))
+		logger.Fatalf("Error building serving clientset: %v", zap.Error(err))
```

Comment:
Thanks for catching this. I believe the right fix should be:
`logger.Fatal("Error building serving clientset", zap.Error(err))`
(removal of %v)

Reply:
done